docs/next-steps.md

```diff
@@ -0,0 +1,51 @@
+# Welcome to the Next Steps for your Kubernetes cluster.
+
+This guide is for [Kubernetes Quick Start](https://aws.amazon.com/quickstart/) admins and users. This guide assumes you have deployed the Kubernetes Quick Start by walking through [Amazon’s Quick Start PDF](https://s3.amazonaws.com/quickstart-reference/vmware/latest/doc/vmware-kubernetes-on-the-aws-cloud.pdf), which uses Amazon’s CloudFormation console.
+
+Now that you have your Kubernetes stack on Amazon, we recommend setting up [WordPress with Helm](http://docs.heptio.com/content/tutorials/aws-qs-helm-wordpress.html) as a demo application to explore your new cluster.
+
+
+> Note:
+>
+>This stack is appropriate for proof of concept (PoC), experimentation, development, and small internal-facing projects. Consider this a test drive.
+>
+>This stack does not currently support upgrades and must be rebuilt for new versions.
+
+
+## Release notes
+Release notes: AWS Quick Start for Kubernetes by VMware
+
+The Quick Start builds Kubernetes 1.13.2.
+
+## Next steps
+If you’ve completed the Kubernetes Quick Start on AWS, we recommend that you try these next steps:
+
+ - Check that your cluster is configured and working as expected by running [Sonobuoy](https://github.com/heptio/sonobuoy) against your cluster. Sonobuoy provides cluster diagnostics by running Kubernetes conformance tests.
+ - Explore your cluster and deploy a demo application: [WordPress with Helm](http://docs.heptio.com/content/tutorials/aws-qs-helm-wordpress.html)
```

Comment:
I think this content is supposed to be in GitHub now?  Same

Reply:
same with the nex tline

Reply:
Yes I haven't yet put those into the repo though. Those will be follow ups (either to this PR or a next) as I get them finished.

Reply:
Cool - follow up PRs fine with me.  I'll do a complete review in the AM